Directory.Build.props

```diff
@@ -17,6 +17,7 @@
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('ILLUMOS'))">illumos</_hostOS>
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('SOLARIS'))">Solaris</_hostOS>
     <_hostOS Condition="$([MSBuild]::IsOSPlatform('WINDOWS'))">windows</_hostOS>
+    <HostOS>$(_hostOS)</HostOS>
     <_hostOS Condition="'$(TargetOS)' == 'Browser'">Browser</_hostOS>
```

Comment:
this can actually be deleted, there's no way we'll be building on a "Browser" host OS I think.

Reply:
we can't just delete it because in case of "browser" the definitions of what is host and what is target does not fit the bill. other build systems and toolchains have a third dimension for it called "build", which we don't support. so here we misuse this (internal) property name in this case.

you can try to delete this line on main to see how it breaks the wasm packaging